src/securityPolicy.c

```diff
@@ -783,6 +946,28 @@ static inline security_policy_t _scriptWitnessPolicy(const bip44_path_t* path, b
 	}
 }
 
+static inline security_policy_t _plutusWitnessPolicy(const bip44_path_t* path, bool mintPresent)
+{
+	switch (bip44_classifyPath(path)) {
+	case PATH_ORDINARY_SPENDING_KEY:
+	case PATH_ORDINARY_STAKING_KEY:
+	case PATH_POOL_COLD_KEY:
+	case PATH_MULTISIG_SPENDING_KEY:
+	case PATH_MULTISIG_STAKING_KEY:
+		WARN_UNLESS(bip44_isPathReasonable(path));
+		SHOW();
+		break;
+
+	case PATH_MINT_KEY:
+		DENY_UNLESS(mintPresent);
+		SHOW();
```

Comment:
I believe `PATH_POOL_COLD_KEY` should be forbidden.

Also, on Trezor we are allowing `PATH_MINT_KEY` in `PLUTUS_SIGNING_MODE` even without `mintPresent`. Not sure whether we agreed on this or the opposite, but we should make it consistent.

And maybe a `break;` missing here (also in `_multisigWitnessPolicy`)?

Reply:
PATH_POOL_COLD_KEY: yes.

PATH_MINT_KEY: I'd say it some kind of misuse to sign with a mint key without a mint present, but it's hard to foresee what Plutus script authors are going to come up with. I'll allow it on Ledger to be consistent with Trezor.